package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "instant-observabilty-site",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Instant Observabilty Site",
```

Looks like I messed up the spelling too!
```suggestion
  "description": "Instant Observability Site",
```